modules/ROOT/pages/tuning-backpressure-maxconcurrency.adoc

```diff
@@ -23,5 +23,7 @@ Similarly, you can limit the number of records that can be processed in a batch
 <batch:job jobName="test-batch" maxConcurrency="${batch.max.concurrency}">
 ----
 
+By default, there is no `maxConcurrency` limit on components that support this parameter. However, the capacity of your system limits concurrency.
```

Comment:
actually, batch has a default maxConcurrency of twice the number of available cores. This is true however for `flow` and `async`.

Reply:
done